packages/backend/src/modules/tournament/tournaments.query.ts

```diff
@@ -1,15 +1,30 @@
-import { Query, Resolver } from 'type-graphql';
+import { Arg, Query, Resolver } from 'type-graphql';
 import { getRepository } from 'typeorm';
 
 import DBTournament from '../../entities/tournament';
 
 import GQLTournament from './tournament';
 
+function dateSort(x: DBTournament, y: DBTournament): number {
+	if (x.createdAt > y.createdAt) {
+		return 1;
+	} else {
+		return 0;
+	}
+}
+
 @Resolver()
 export default class TournamentsQueryResolver {
 	@Query(() => [GQLTournament])
-	async tournaments(): Promise<GQLTournament[]> {
+	async tournaments(@Arg('sort', { nullable: true }) sort?: boolean): Promise<GQLTournament[]> {
```

Comment:
would it be better to make the orderBy field be an input type containing the field to order by and whether to order in ascending or descending order?